pkg/markers/terrajet.go

```diff
@@ -0,0 +1,59 @@
+package markers
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	markerPrefixTerrajet = "+terrajet:"
+
+	errFmtCannotParse = "cannot parse as a terrajet prefix: %s"
+)
+
+var (
+	markerPrefixCRDTFTag   = fmt.Sprintf("%scrdfield:TFTag=", markerPrefixTerrajet)
```

Comment:
```suggestion
	markerPrefixCRDTFTag   = fmt.Sprintf("%scrd:field:TFTag=", markerPrefixTerrajet)
```

Would `+terrajet:crd:field:TFTag` be more open to expansions in the future, like `+terrajet:crd:type:NameOverride`?

Reply:
We could change it later since it is no longer part of our API (private parameter of the package), but doing it now also sounds good.